specification/resources/resource-manager/Microsoft.Resources/stable/2019-10-01/resources.json

```diff
@@ -4009,8 +4012,9 @@
         "tags": [
           "Tags"
         ],
-        "operationId": "Tags_ResourceCreate",
-        "description": "Create or Replace existing tags with passing in tags.",
+        "operationId": "Tags_CreateOrReplaceTagsOnObject",
```

Comment:
>CreateOrReplace [](start = 29, length = 15)

we have been using the term CreateOrUpdate for "PUT" calls. do we want to follow that?

Reply:
Updated